### DISCUSSION TITLE
Users in WA - Puget Sound (Seattle Tacoma Everett)

### DISCUSSION THREAD

**Discussion by TerryFrench** [meshtastic/Discussions#131] [North America]
Migration from [discourse](https://meshtastic.discourse.group/t/user-in-seattle-tacoma-wa/8638 ) “Users in Seattle - Tacoma - WA”

**Comment by TerryFrench** [comment]
Hi folks, I found the discourse and created this as a continuation.  I have a few nodes to play wit and plan to put one on a mast at home (but bad location no los) and hopefully one at work as I got a couple enclosures. Solar panels and one dipole fiberglass antenna. 

I’m curious what MQTT servers and topics we might have/use for our area. 

Thanks.

**Comment by kusuriya** [comment]
I run a node out near Milton, and have been building more nodes for funsies.
